04-spatial-operations.Rmd

```diff
@@ -41,53 +46,52 @@ It is important to note that spatial operations that use two spatial objects rel
 ## Spatial subsetting
 
 Spatial subsetting is the process of selecting only those features of a spatial object that in some way *intersect* with another spatial object.
+<!-- can we already reference the intersection figure here? Generally asking, why can't we present the other topological operations here? Would fit nicely in this section... -->
```

Comment:
Yes, I wouldn't separate spatial subsetting and the other topological operations. They just fit fine here. Moreover, we need also need them for spatial joins so we should present them here already! Maybe we have then to rename the header of the subsection.

Reply:
From a teaching perspective it's much easier to learn things with one operation first and then discover there are many you can use. A cross-reference here would be ideal.

Reply:
I agree. But I meant to just append the topological stuff to the spatial subsetting and not separate it into two subsections with other subsections in between.